pkg/util/leaktest/leaktest.go

```diff
@@ -58,8 +58,9 @@ func interestingGoroutines() map[int64]string {
 			strings.Contains(stack, "sentry-go.(*HTTPTransport).worker") ||
 			// Ignore the opensensus worker, which is created by the event exporter.
 			strings.Contains(stack, "go.opencensus.io/stats/view.(*worker).start") ||
-			// Ignore pgconn which creates a goroutine to do an async cleanup.
+			// Ignore pgconn which creates two goroutines to do async cleanup.
 			strings.Contains(stack, "github.com/jackc/pgconn.(*PgConn).asyncClose.func1") ||
+			strings.Contains(stack, "github.com/jackc/pgconn/internal/ctxwatch.(*ContextWatcher).Watch.func1()") ||
```

Comment:
what is this about? just want to check its intentional

Reply:
we should be very sure this isn't a missing Close() before we add it to the list or we could be hiding a real drain / shutdown bug

Reply:
good call. i'll investigate further and skip the flaky test for now.